configure.ac

```diff
@@ -1678,6 +1673,26 @@ yes)
     fi
     AC_MSG_RESULT($ac_cv_enable_unreachable_code_warning)
 
+    AC_MSG_CHECKING(if we can turn on $CC strict-prototypes warning)
+     ac_save_cc="$CC"
+     CC="$CC -Wstrict-prototypes"
```

Comment:
Is now `-Werror` needed?

Reply:
Hmm, some other `-Wxxx` checks doesn't add `-Werror`.

Anyway, I can't test this checking properly with g++, because g++ doesn't support `-std=c99` too.
May I remove this check and simply add `-Wstrict-prototypes` in addition to `-std=c99`?

Reply:
```
$ g++ -Wstrict-prototypes -c x.c
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
$ echo $?
0

$ g++  -Wstrict-prototypes -Werror -c x.c
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
$ echo $?
0

$ g++ -Werror -Wstrict-prototypes -c x.c
cc1plus: error: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++ [-Werror]
cc1plus: all warnings being treated as errors
$ echo $?
1
```

It looks `-Werror` is needed before `-Wstrict-prototypes`.